lib/read_parsers.cc

```diff
@@ -41,6 +41,7 @@ SeqAnParser::SeqAnParser( char const * filename ) : IParser( )
     }
     __asm__ __volatile__ ("" ::: "memory");
     _private->seqan_spin_lock = 0;
+    _num_reads = 0;
```

Comment:
Will the constructor be inherited like that? I never got my head around ctor inheritance in C++ to be honest. My understanding was that it doesn't automatically, but I'm probably wrong. In which case yes, it's redundant.